proposal/comp110_architecture_proposal.tex

```diff
@@ -18,60 +18,60 @@ \section*{Topic}
+\item[Web link:] \url{http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=7317906&newsearch=true&queryText=procedural\%20level\%20generation\%20for\%20a\%202D\%20platform\%20game}
+\item[Full text link:] \url{http://www.eecs.harvard.edu/~gal/Papers/CIG_2015_submission_24.pdf}
+\item[Comments:] This is a very recent article with many strong points and ideas on how procedurally generated levels can react to player feedback.
 \end{description}
 
 \section*{Paper 2}
 \begin{description}
-\item[Title:] Title of paper
-\item[Citation:] \cite{bibtex_key}
-\item[Abstract:] Copy and paste the abstract here
-\item[Web link:] Give the URL of the paper in IEEE Xplore, ACM Digital Library, or similar
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
+\item[Title:] Towards Automatic Personalized Content Generation for Platform Games
+\item[Citation:] \cite{Shaker}
+\item[Abstract:] ``In this paper, we show that personalized levels can be auto
+matically generated for platform games.  We build on previous work, where models were derived that predicted player experience based on features of level design and on playing styles.  These models are constructed using preference learning,  based  on  questionnaires  administered  to  players after playing different levels.  The contributions of the current paper are (1) more accurate models based on a much larger data set; (2) a mechanism for adapting level design parameters to given players and playing style; (3) evaluation of this adaptation mechanism using both algorithmic and human players. The results indicate that the adaptation mechanism effectively optimizes level design parameters for particular players.''
+\item[Web link:] \url{http://iris.ofai.at:7777/iris_db/index.php/publications/show/348}
+\item[Full text link:] \url{http://www.aaai.org/ocs/index.php/AIIDE/AIIDE10/paper/viewFile/2135/2546}
+\item[Comments:] This paper shows that a randomly generated level can be personalised for a player automatically. The paper has also undergone 5 years of peer review and has been cross referenced 42 times.
 \end{description}
 
 \section*{Paper 3}
 \begin{description}
-\item[Title:] Title of paper
-\item[Citation:] \cite{bibtex_key}
-\item[Abstract:] Copy and paste the abstract here
-\item[Web link:] Give the URL of the paper in IEEE Xplore, ACM Digital Library, or similar
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
+\item[Title:] A framework for analysis of 2D platformer levels
+\item[Citation:] \cite{Smith}
```

Comment:
Supporting Material. Look at papers which cite this one for algorithms based on this analytical approach.